hack/verify-govet.sh

```diff
@@ -21,8 +21,7 @@ FAILURE=false
 test_dirs=$(find_files | cut -d '/' -f 1-2 | sort -u)
 for test_dir in $test_dirs
 do
-  go tool vet -shadow=false \
-              $test_dir
+  GOPATH=$PWD/_output/local/go:$PWD/Godeps/_workspace:$GOPATH go tool vet -shadow=false $test_dir
```

Comment:
Not sure if `$PWD` is safe to use here- do other scripts make assumptions about where the script is run from? I used it in my testing but I assumed there was some other pattern in place or utility to resolve the local output paths for us.